### PR TITLE
ci: hide nextest progress bar

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

The log for `unit test (deterministic)` is weird https://buildkite.com/risingwavelabs/main/builds/5487#018a477b-acd5-44ae-a7dd-b45ca09f9774

Raw log contains a lot of `RUNNING` output.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: ci/scripts/common.sh [hunk 1]
```diff
@@ -13,6 +13,8 @@ export CARGO_MAKE_PRINT_TIME_SUMMARY=true
 export MINIO_DOWNLOAD_BIN=https://ci-deps-dist.s3.amazonaws.com/minio
 export MCLI_DOWNLOAD_BIN=https://ci-deps-dist.s3.amazonaws.com/mc
 export GCLOUD_DOWNLOAD_TGZ=https://ci-deps-dist.s3.amazonaws.com/google-cloud-cli-406.0.0-linux-x86_64.tar.gz
+export CI=true
```
Comment (xxchan) [inline ci/scripts/common.sh:16]: https://github.com/nextest-rs/nextest/blob/6e00ddeb62229c753bf346465fbb49e093bd2f3e/nextest-runner/src/reporter.rs#L267-L274

https://github.com/zkat/is_ci/blob/bc4fda761a9810a0e967d1470772fedd78d79360/src/lib.rs#L26-L28

Comment (xxchan) [reply ci/scripts/common.sh:16]: Not sure if it's needed, but I think it doesn't hurt.

Comment (xxchan) [reply ci/scripts/common.sh:16]: weird

image

**Comment by BugenZhao** [comment]
Does this also mean that we don't need the `tee` hack anymore?

**Comment by xxchan** [comment]
I think so

**Review by BugenZhao — APPROVED** [review]

**Pull request merged**